CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ see if you need to change any of these:
 - Added new SDA workflow for the 342 North America anchor sites.
 - Added new feature of preparing initial conditions for MODIS LAI, AGB, ISCN SOC, and soil moisture across NA anchor sites.
 - Added GEDI AGB preparation workflow.
+- Added new feature of downloading NASA DAAC ORNL database.
```

Comment:
I think this log message needs to be more informative as you're not downloading the entire ORNL DAAC database

Reply:
Fixed.